docs/token/behaviour.md

```diff
@@ -0,0 +1,35 @@
+# Aragon Network Token
+
+**👉 Important information**:
+
+- ANT's token contract address is [`0x960b236A07cf122663c4303350609A66A7B288C0`](https://etherscan.io/token/0x960b236A07cf122663c4303350609A66A7B288C0)
+- ANT is a ERC20 token, specifically a [MiniMe token](../developers/minime.md)
+- Its current controller is a [pass-through placeholder](#controller), meant to be replaced for the [Aragon Network](https://aragon.network/)
+
+## Introduction
+
+The Aragon Network Token ("ANT") is an ERC20-compliant token deployed to Ethereum mainnet. It was launched through the [Aragon Network Token sale](https://aragon.org/blog/announcing-the-aragon-network-token-sale-fe83fe36902c) in May, 2017.
+
+In its current configuration, the token's behaviour mimics a vanilla ERC20 token:
+
+- Its supply, and holders' balances, are constant
+- All transfers are allowed, with no internal fee mechanisms
+- No functionality for blacklisting addresses
+
+In addition to the vanilla behaviour, it implements a number of features that are useful for the Aragon Network:
+
+- Historical token balance records
+- Cloning, with balances from a specific block snapshot
+- An optional controller (see below)
+
+And finally, a specific behaviour implemented solely for the token sale:
+
+- Irrevocable time-based vesting, callable only prior to the token sale starting
```

Comment:
This functionality wasn't used just for the sale (although the sale contract used it). The AA multisig can still whitelist addresses that can transfer tokens with vesting. We should probably shut this down now as all tokens are now fully vested as long as the ANT contract is concerned and using vesting contracts is clearly a best practice now.

The AA multisig would need to:
- Use `ant.setCanCreateGrants(addr, false)` for all the accounts that can create vesting grants right now. I think only the sale contract and an account I control (`0x4838`) are currently whitelisted. 
- `ant.changeVestingWhitelister(address(0))`

Reply:
Good catch. Since there are no events emitted for these, we may want to use a subgraph with call handlers to double check we know all the addresses cc @facuspagnuolo.